test/system/flows/student/exercise/views_dig_deeper_test.rb

```diff
@@ -0,0 +1,82 @@
+require "application_system_test_case"
+require_relative "../../../../support/capybara_helpers"
+
+module Flows
+  module Student
+    module Exercise
+      class ViewsDigDeeperTest < ApplicationSystemTestCase
+        include CapybaraHelpers
+
+        test "view dig deeper section by unlocking via unlock button" do
+          user = create :user
+          track = create :track
+          create :user_track, user: user, track: track
+          create :hello_world_solution, :completed, track: track, user: user
+          exercise = create :practice_exercise, track: track, slug: 'hamming'
+          create :exercise_approach, exercise: exercise
+          create :exercise_article, exercise: exercise
+          exercise.update(has_approaches: true)
+
+          use_capybara_host do
+            sign_in!(user)
+            visit track_exercise_path(track, exercise)
+
+            assert_text "Dig Deeper"
+            find("span", text: "Dig Deeper").ancestor("div[data-tooltip-type=\"automation-locked\"]").hover
+            assert_text "You can only view the Dig Deeper tab for this exercise if you've attempted this exercise at least once."
+
+            # Simulate having passing tests
+            solution = create :practice_solution, user: user, exercise: exercise, downloaded_at: Time.zone.now
```

Comment:
```suggestion
            solution = create :practice_solution, user: user, exercise: exercise, downloaded_at: Time.current
```

(We use this everywhere else)